backend/corpora/common/utils/authorization_checks.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+
+def has_scope(required_scope: str, scopes: Union[list, str]) -> bool:
+    result = False
+    _scopes = scopes.split(" ") if isinstance(scopes, str) else scopes
+    for scope in _scopes:
+        if scope == required_scope:
+            result = True
+    return result
```

Comment:
```suggestion
    _scopes = scopes.split(" ") if isinstance(scopes, str) else scopes
    return True if required_scope in _scopes else False
```

Reply:
What's not to love about a function that comprises two one-line if-else statements? 🤠